api/src/main/java/org/apache/iceberg/Table.java

```diff
@@ -308,6 +308,13 @@ default UpdateStatistics updateStatistics() {
   /** Returns a {@link LocationProvider} to provide locations for new data files. */
   LocationProvider locationProvider();
 
+  /**
+   * Returns the current statistics files for the table
+   *
+   * @return the current statistics files for the table
+   */
+  List<StatisticsFile> statisticsFiles();
```

Comment:
Do we want to expose these directly? What about instead adding an API for finding statistics? That way consumers don't need to decide which one to use themselves.

Reply:
> Do we want to expose these directly? 

Yes, these should be exposed, just like we expose other information about table (files, their min/maxes, etc.) _and_ we provide convenience functionality on top of them (like `planFiles`).

> What about instead adding an API for finding statistics?

Good idea. What would you like to see here?